src/app/zap-templates/zcl/zcl.json

```diff
@@ -422,7 +415,13 @@
             "DSTOffsetListMaxSize"
         ],
         "Temperature Control": ["SupportedTemperatureLevels"],
-        "Dishwasher Mode": ["SupportedModes", "CurrentMode", "FeatureMap"],
+        "Dishwasher Mode": [
+            "SupportedModes",
+            "CurrentMode",
+            "StartUpMode",
+            "OnMode",
```

Comment:
This cluster does not have StartUpMode or OnMode attributes.  But more importantly: why is this being changed?